test/setTimerTest.py

```diff
@@ -0,0 +1,29 @@
+from base.DriverClass import Driver
+from pages.setTimerPage import SetTimer
+from utils.NotificationUtil import Notification
+from utils.SwipeUtil import SwipeUtility
+
+
+def test_count_down():
+
+    driver1 = Driver()
+    driver = driver1.getDriverMethod()
+
+    set_timer = SetTimer(driver)
+    swipe_utility = SwipeUtility(driver)
+    notification = Notification(driver)
+
+    set_timer.clickPermissionsButton()
+    set_timer.clickAllowNoifications()
+    set_timer.clickGotItBtn()
+    set_timer.clickCountDownBtn()
+    assert set_timer.isFrameDisplayed(), "FrameLayout not found."
+    swipe_utility.swipe_action(swipe_duration=100)
```

Comment:
Here you need to set specific amount of time. For ex. you need to have opportunity to set 5 sec, 8sec, 32sec, any time. And after that you need to check that notification appears in a second after this time. 
Now you just set random time and just wait for notification. Notification can appears earlier, 5 sec later - your test passed.